mcs/class/System.Security/System.Security.Cryptography.Pkcs/Pkcs9SigningTime.cs

```diff
@@ -78,7 +78,7 @@ public override void CopyFrom (AsnEncodedData asnEncodedData)
 				throw new ArgumentNullException ("asnEncodedData");
 
 			Decode (asnEncodedData.RawData);
-			Oid = asnEncodedData.Oid;
+			(this as AsnEncodedData).Oid = asnEncodedData.Oid;
```

Comment:
`((AsnEncodedData)this).Oid` otherwise you will get a `NullReferenceException` if the cast fails.

Reply:
I will update it. I actually copied it from the same file, so it is present on two other places.